Misc. updates for new repositories and releasing stuff

aiyengar2 opened the pull request:
Used for the RKE2 charts migration

cbron commented on README.md:
```diff
@@ -5,17 +5,20 @@ charts-build-scripts
+#### If you are creating a new charts repository 
 
-Set the environment variable `GITHUB_AUTH_TOKEN` to your [personal Github Access Token](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token).
+Checkout into the Git branch that corresponds to your Source, Staging, or Live branch.
```
nit: "Checkout into the" -> "Checkout the"

aiyengar2 replied:
will update, good catch.

cbron commented on templates/template/scripts/regenerate-packages:
```diff
@@ -3,6 +3,13 @@ set -e
+if [[ -z ${BRANCH} ]]; then
+    branch=$(git rev-parse --abbrev-ref HEAD)
+else
+    echo "Using branch ${BRANCH}"
```
Would think this echo should be outside of conditional so it always prints ?

aiyengar2 replied:
Sure, will update.

cbron commented on init.sh:
```diff
@@ -0,0 +1,41 @@
+    exit 1
+fi
+
+# cd into the Git repository that you are converting into a charts repository
```
this comment looks out of place

aiyengar2 replied:
ack, I left that behind on accident. Removed.

cbron commented on init.sh:
```diff
@@ -0,0 +1,41 @@
+    echo "You will need to run make docs manually after filling in the configuration.yaml"
+    echo "To add a template for Github Workflow based pull-requests, run the following script and update .github/workflows/pull-request.yaml manually"
+    echo "curl ${CBS_RAW_LINK}/templates/.github/workflows/pull-request.yaml --output .github/workflows/pull-request.yaml"
+     echo "To add a template for Github Workflow based automatic pushes, run the following script and update .github/workflows/push.yaml manually"
```
nit: extra space

aiyengar2 replied:
ack, updated

cbron reviewed (commented):
Don't fully comprehend the init.sh stuff, but I get what you are going for

aiyengar2 commented:
Closing this in favor of https://github.com/rancher/charts-build-scripts/pull/24, which is based off of my fork.

@cbron, I've addressed all the comments here in the other PR.
